libkmod/libkmod-elf.c

```diff
@@ -178,27 +178,28 @@ static inline const void *elf_get_mem(const struct kmod_elf *elf, uint64_t offse
 	return elf->memory + offset;
 }
 
-static inline const void *elf_get_section_header(const struct kmod_elf *elf, uint16_t idx)
+/* returns offset to section header for section with given index or 0 on error. */
```

Comment:
returning 0 as error is odd and we should explain why it's fine here:  no section should have offset == 0 as there's at least the elf header too.

Reply:
Added to comment